Formula/couchdb.rb

```diff
@@ -78,37 +123,74 @@ def post_install
     end
   end
 
-  def caveats; <<-EOS.undent
+  def caveats
+    s = <<-EOS.undent
     To test CouchDB run:
         curl http://127.0.0.1:5984/
 
     The reply should look like:
         {"couchdb":"Welcome","uuid":"....","version":"#{version}","vendor":{"version":"#{version}-1","name":"Homebrew"}}
     EOS
+    s += "\n#{geocouch_caveats}" if build.with? "geocouch"
+    s
   end
 
-  plist_options :manual => "couchdb"
+  def geocouch_caveats; <<-EOS.undent
+    You can access, test, and configure your new geocouch with:
+
+      http://127.0.0.1:5984
+      http://127.0.0.1:5984/_utils/couch_tests.html?script/couch_tests.js
+      http://127.0.0.1:5984/_utils
+
+    To uninstall geocouch from your couchdb installation, uninstall couchdb
+    and re-install it without the '--with-geocouch' option.
+
+      brew uninstall couchdb
+      brew install couchdb
 
-  def plist; <<-EOS.undent
-    <?xml version="1.0" encoding="UTF-8"?>
-    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-    <plist version="1.0">
-    <dict>
-      <key>KeepAlive</key>
-      <true/>
-      <key>Label</key>
-      <string>#{plist_name}</string>
-      <key>ProgramArguments</key>
-      <array>
-        <string>#{opt_bin}/couchdb</string>
-      </array>
-      <key>RunAtLoad</key>
-      <true/>
-    </dict>
-    </plist>
+    To see these instructions again, just run 'brew info couchdb'.
```

Comment:
I killed all the remaining caveats, too, since they were just instructions on how to test couchdb, and that should go in, well, the `test` definition. Converted them to actual `test` steps so now there's no caveats, and the test does more checking.